example_sns_init.yaml

```
@@ -25,104 +31,307 @@ description: >
+
+    # The wait for quiet algorithm extends the voting period of a proposal when
+    # there is a flip in the majority vote during the proposal's voting period.
+    #
+    # Without this, there could be an incentive to vote right at the end of a
+    # proposal's voting period, in order to reduce the chance that people will
+    # see and have time to react to that.
+    #
+    # TODO: fix text below
+    # If this value is set to 1 day, then a change in the majority vote at the
+    # end of a proposal's original voting period results in being extended by an
+    # additional day. Another change at the end of the extended
+    # period will cause the voting period to be extended by half a day, and so on.
+    #
+    # The total extension to the voting period will never be more than twice this
+    # value.
+    #
+    # For more information, please refer to
+    # https://wiki.internetcomputer.org/wiki/Network_Nervous_System#Proposal_decision_and_wait-for-quiet
     maximum_wait_for_quiet_deadline_extension: 1 day
 
-# It is fine to leave these alone.
+# Configuration of SNS neuron-related fields.
 Neurons:
+    # The minimum amount of SNS tokens an SNS ledger account must have to stake
+    # a neuron. The larger the minimum, the more tokens must be acquired to be
+    # able to vote.
     minimum_creation_stake: 1 tokens
 
-# It is fine to leave these alone.
+# Configuration of voting-related fields of the SNS. These fields have no effect
+# on the NNS proposal created by this file, but will affect the SNS itself.
 Voting:
+    # The minimum dissolve delay a neuron must have to be able to cast votes on
+    # proposals. The higher the minimum, the longer users will need to be
+    # aligned with the SNS to vote. However, the longer the minimum, the longer
+    # users will not be able to access the SNS tokens used to stake the neuron.
     minimum_dissolve_delay: 1 month
 
+    # Configuration of the various voting power bonuses that are applied to
+    # neurons to incentivize alignment with the best interest of the DAO.
     MaximumVotingPowerBonuses:
+        # Users with a higher a dissolve delay are incentivized to take the
+        # long-term interests of the SNS into consideration when voting. To
+        # reward this long-term commitment, this bonus can be set to a
+        # percentage greater than zero, which will result in neurons having
+        # their voting power increased in proportion to their dissolve delay.
+        #
+        # If you do not want this bonus should to be applied for neurons with
+        # higher dissolve delay, set `bonus` to `0%` and those neurons will not
+        # receive higher voting power.
         DissolveDelay:
+            # This parameter sets the maximum dissolve delay a neuron can have.
+            # When reached, the maximum dissolve delay bonus will be applied.
             duration: 8 years
+            # If a neuron's dissolve delay is `duration` or older, its voting
```

```suggestion
            # If a neuron's dissolve delay is `duration`, its voting
```
`duration` should be the max possible value => the dissolve delay cannot be higher (or "older")